verticapy/sql/functions/regexp.py

```diff
@@ -170,17 +172,18 @@ def regexp_instr(
         titanic = load_titanic()
         # Apply the regexp_instr function, creating a new "title_start" column
         titanic["title_start"] = vpf.regexp_instr(titanic["name"],
-                                                 '([A-Za-z])+\. ',
+                                                 '([A-Za-z])+\\.',
```

Comment:
are you sure?

Reply:
It seems that VerticaPy does not like the single backslash:
![Capture](https://github.com/vertica/VerticaPy/assets/46414488/7a1e6335-e656-47a9-8a21-a1d302887fcb)